vscode-antimony/src/server/main.py

```diff
@@ -141,6 +145,30 @@ def sbml_file_to_ant_file(ls: LanguageServer, args):
             'file': full_path_name
         }
 
+@server.thread()
+@server.command('antimony.sendType')
+def get_type(ls: LanguageServer, args):
+    global antfile_cache

```

Comment:
Why are you using globals? Globals complicate extending and testing code, and generally are a bad practice.